src/framerate_gui.cpp

```diff
@@ -1077,11 +1077,11 @@ void ConPrintFramerate()
 	for (PerformanceElement e = PFE_FIRST; e < PFE_MAX; e++) {
 		auto &pf = _pf_data[e];
 		if (pf.num_valid == 0) continue;
-		const char *name;
+		std::string_view name;
 		if (e < PFE_AI0) {
 			name = MEASUREMENT_NAMES[e];
 		} else {
-			seprintf(ai_name_buf, lastof(ai_name_buf), "AI %d %s", e - PFE_AI0 + 1, GetAIName(e - PFE_AI0)),
+			ai_name_buf = fmt::format("AI {} {}", e - PFE_AI0 + 1, GetAIName(e - PFE_AI0));
```

Comment:
Is using `ai_name_buf` still really needed ?

Reply:
If we do not want to copy `MEASUREMENT_NAMES[e]` into a `std::string` each time, then yet. Otherwise `name` could become `std::string` with a technically unneeded copy. After all, the `std::string` must outlive the `std::string_view`.